dev/provider_packages/refactor_provider_packages.py

```diff
@@ -730,6 +730,9 @@ def kubernetes_package_filter(node: LN, capture: Capture, filename: Filename) ->
             .rename("add_sidecar")
         )
 
+    def refactor_www(self):
+        (self.qry.select_module("airflow.www").rename("airflow.www_rbac"))
```

Comment:
I wonder if we shouldn't have moved this module back before the Airflow 2.0 release. This is a breaking change and has very little benefit.  Moving everything from airflow.www_rbac to airflow.www makes it difficult to write universal plugins.

Reply:
I think even if we do - this is a separate discussion.